_posts/2024-04-30-openttd-14-1.md

```diff
@@ -0,0 +1,13 @@
+---
+title: OpenTTD 14.1
+author: michi_cc
+---
+
+While OpenTTD 14 has been well received (just as we've recieved a lot of survey data), it has apparently also been occasionally out-of-sync with itself.
```

Comment:
```suggestion
Our players have played a lot of OpenTTD 14 since it was released a few weeks ago -- in last week alone those who opted into our [survey](https://www.openttd.org/news/2024/03/16/survey-https-infra) played 34,700 hours across 17,219 games.
```